### ISSUE TITLE
Error: 'NoneType' object is not callable

### ISSUE THREAD

**Issue by level-zer0** [threatexpress/domainhunter#9]
I have ran the following commands:
- ./domainhunter.py -k <keyword> -r 1000 -c
- python domainhunter.py -k <keyword> -r 1000 -c
- python3 domainhunter.py -k <keyword> -r 1000 -c

There are two separate issues:

**Error #1**
('[!] Error: ', AttributeError("'NoneType' object has no attribute 'find_all'",))

![image](https://user-images.githubusercontent.com/41021765/52668474-42805b80-2ed9-11e9-8aec-89369f6b7256.png)

I changed "findAll" to "find_all" as the previous syntax was for BeautifulSoup3 and it is now importing bs4. After trying to run the command again, I stumbled upon the next error.

**Error #2**
![image](https://user-images.githubusercontent.com/41021765/52668561-84110680-2ed9-11e9-84b2-893ea4d09bb8.png)

The assumption is that there is an error within the query, but I have not had much fixing it. Was wondering if you had  a quick fix for it. Thank you!

**Comment by andrewchiles** [comment]
Hey there, thanks for posting this issue! I haven't had a chance to test this out yet, but I assume the HTML format returned by expireddomains.net has changed (fairly common) and broke the results table parsing. We will look into a fix.

**Comment by andrewchiles** [comment]
Quick Update - your error is based on a keyword search that returns no results. It appears the tool is still functioning properly when a keyword with valid results is returned. If you see that error, assume no results were returned for the keyword. You can manually verify this by going to www.expiredomains.net and typing your keyword to confirm no returned results.

**Comment by level-zer0** [comment]
I appreciate the quick response and help! It may not be necessary, but I created an account on expireddomains.net, logged in, and it worked fine. For whatever reason, expireddomains.net was not letting me query both when I ran your tool and typed in manually.

While this may induce false positives, I'd be interested in the future to stop querying upon error. There were only 30 some hits, but it kept querying because I specified the results flag to 1000. Will close this issue. Thanks again!
